spec/spec_helper.rb

```diff
@@ -1 +1,23 @@
+if ENV['COVERAGE']
+  # In order to archive an expected coverage analysis we need to eager load
+  # all logstash code base, otherwise it will not get a good analysis.
+
+  require "pluginmanager/main"
+  Dir.glob("lib/logstash/patches/*.rb") do |file|
+    require file
+  end
+
+  Dir.glob("lib/logstash/*.rb") do |file|
+    require file
+  end
+
+  Dir.glob("lib/logstash/util/*.rb") do |file|
+    next if ["lib/logstash/util/prctl.rb"].include?(file)
+    require file
```

Comment:
because i tend to write like this in case we've more than one file in the feature, I can change this for now and make it just an string comparison.